examples/openstack/.kitchen.yml

```diff
@@ -0,0 +1,37 @@
+---
+driver:
+  name: terraform
+
+provisioner:
+  name: terraform
+  # The number of seconds to wait for the Terraform apply command to be
+  # successful before raising an error.
+  apply_timeout: 1000
+  color: true
```

Comment:
The `color` and `directory` values are redundant since they use the default values.